### DISCUSSION TITLE
Navbar Brand is centered

### DISCUSSION THREAD

**Discussion by MohamadSalman11** [twbs/bootstrap#40906] [Q&A] [answered]
Why should the navbar brand be centered even when there’s space on the left side and no other elements present?

Original Behavior:
![chrome_UwUysRKorh](https://github.com/user-attachments/assets/76a3d511-b932-43dd-af8a-249e91138add)

My Suggestion:
![chrome_6xqGfrmi30](https://github.com/user-attachments/assets/3e5e10db-219d-4c65-9456-d44919b38fee)

**Comment by julien-deramond** [comment, marked as answer]
Hey @MohamadSalman11 
It was a voluntary design choice when it has been created. I don't have the details.
